packages/apps-config/src/endpoints/testingRelayPaseo.ts

```diff
@@ -249,7 +249,20 @@ export const testParasPaseo: Omit<EndpointOption, 'teleport'>[] = [
     text: 'Xcavate',
     ui: {
       color: '#FF0083',
-      logo: chainsXcavatePNG
+      logo: chainsNewxcavPNG
+    }
+  },
+  {
+    homepage: 'zeitgeist.pm',
```

Comment:
It's saying this is a duplicate, when you were fixing conflicts it looks like you fixed them wrong.